### PR TITLE
added support for gh actions

### PR DESCRIPTION
**Pull request by cdietrich**
added support for gh actions
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: scripts/integrationtests-wizard.sh [hunk 1]
```diff
@@ -1,5 +1,6 @@
 cd "$(pwd)/integrationtests"
-
+TRAVIS_BUILD_DIR="${TRAVIS_BUILD_DIR:-$GITHUB_WORKSPACE}"
```
Comment (cdietrich) [inline scripts/integrationtests-wizard.sh:2]: TODO we will remove this when we are sure we can ditch travis completely

**Review by ArneDeutsch — APPROVED** [review]
Looks ok to me, but I guess it would be good if @kthoms would have a look as well.

File: scripts/integrationtests-wizard.sh (hunk 1 above)
Comment (NicoPrediger) [reply scripts/integrationtests-wizard.sh:2]: travis.yml is deleted, shouldn't this be removed now, then?

other than that: LGTM

**Review by NicoPrediger — APPROVED** [review]

File: scripts/integrationtests-wizard.sh (hunk 1 above)
Comment (cdietrich) [reply scripts/integrationtests-wizard.sh:2]: we still dont know where we will go to

**Pull request merged**